build/main.js

```diff
@@ -19,7 +19,7 @@ var __copyProps = (to, from, except, desc) => {
 };
```

Comment:
issue: Including build files in the git history. 
This should be ignored. Let me know the reason if not.

Reply:
You're right, I'll fix it as soon as I get to the keyboard

Reply:
Done, see the latest commit :)